<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListLinksDialogClass</class>
 <widget class="QDialog" name="ListLinksDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListLinksDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTableWidget" name="linksTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Width</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Height</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="editButton">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
             <property name="icon">
              <iconset resource="../Resources/MainWindow.qrc">
               <normaloff>:/MainWindow/icons/tinycolor/icons8-edit-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-edit-16.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="../Resources/MainWindow.qrc">
               <normaloff>:/MainWindow/icons/tinycolor/icons8-trash-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-trash-16.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Close</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ListLinksDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
